universal-login-react/src/core/services/DeleteAccountService.ts

```diff
@@ -0,0 +1,41 @@
+import {utils} from 'ethers';
+import {transactionDetails} from '../../core/constants/TransactionDetails';
+import {DeployedWallet} from '@universal-login/sdk';
+
+type ErrorsType = {
+  usernameError: boolean;
+  verifyFieldError: boolean;
+};
+
+type InputsType = {
+  username: string,
+  verifyField: string
+};
+
+const validateUsername = (usernameFromInput: string, usernameLogged: string): boolean =>
+  usernameFromInput === usernameLogged;
+
+export const validateVerifyField = (verifyFieldFromInput: string): boolean => {
+  const verifyFieldText = 'DELETE MY ACCOUNT';
+  return verifyFieldFromInput === verifyFieldText;
+};
+
+const checkInputsAgainstError = (usernameLogged: string, inputs: InputsType) => ({
+  usernameError: !validateUsername(inputs.username, usernameLogged),
+  verifyFieldError: !validateVerifyField(inputs.verifyField)
+});
+
+export const getInputClassName = (inputError: boolean) => inputError ? 'delete-account-input-error' : 'delete-account-input';
```

Comment:
This code deals with UI (HTML and React), it shouldn't be in services

Reply:
This is great example for the famous Application layer :) between UI and domain.

Reply:
https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition